src/components/Page/LandingPageWidgets/MembersWidget.vue

```diff
@@ -118,9 +118,10 @@ export default {
 
 		updateShowMembersCount: debounce(function() {
 			// How many avatars (44px width + 12px gap) fit? Substract one for the more button.
-			const membersWidth = this.$refs.members?.clientWidth
-			if (membersWidth) {
-				const maxMembers = Math.floor(membersWidth / 56) - 1
+			const parentWidth = this.$refs.members?.parentElement?.clientWidth
+			const siblingWidth = this.$refs.members?.nextElementSibling?.clientWidth
+			if (parentWidth && siblingWidth) {
+				const maxMembers = Math.floor((parentWidth - (siblingWidth + 20)) / 56) - 1
```

Comment:
This only works if there is only one sibling following the members widget, right?

Do we really need to rely on that or could we make the members ref always fill the available space and measure it instead?

Reply:
Indeed, much better. The required fix turned out to be a simple `flex-grow: 1` on the right spot :magic_wand: